File: ruby/easy/sum_of_square_numbers.rb
# @param {Integer} c
# @return {Boolean}
def judge_square_sum(c)
    sqrt = Math.sqrt(c)
    for i in 0..sqrt
        rem = (c - (i ** 2))
        return true if (Math.sqrt(rem) % 1) == 0
    end
    return false
end

File: javascript/easy/sumOfSquareNumbers.js
/**
 * @param {number} c
 * @return {boolean}
 */
var judgeSquareSum = function(c) {
    let sqrt = parseInt(Math.sqrt(c));
    for(let i = 0; i <= sqrt; i++) {
        let rem = (c - (i * i));
        if((Math.sqrt(rem) % 1) === 0) {
            return true;
        }
    }
    return false;
};
